File: ympa_fsharp_giraffe_server/src/model/TimePeriodDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.TimeUnitType

module TimePeriodDTO =

  //#region TimePeriodDTO


  type TimePeriodDTO = {
    TimePeriod : int;
    TimeUnit : TimeUnitType;
    Comment : string;
  }
  //#endregion

File: src/main/scala/org/openapitools/server/model/TimePeriodDTO.scala
package org.openapitools.server.model


/**
 * Временной отрезок с комментарием. Требования к содержанию комментария зависят от контекста использования параметра и указаны в описании поля, которое его содержит.
 *
 * @param timePeriod Продолжительность в указанных единицах. for example: ''null''
 * @param timeUnit  for example: ''null''
 * @param comment Комментарий. for example: ''null''
*/
final case class TimePeriodDTO (
  timePeriod: Int,
  timeUnit: TimeUnitType,
  comment: Option[String] = None
)

